ddp-workspace/projects/ddp-mbc/src/app/app.module.ts

```diff
@@ -73,7 +73,7 @@ tkCfg.cBioPortalLink = 'http://www.cbioportal.org/study?id=brca_mbcproject_wagle
 tkCfg.countMeInUrl = 'https://joincountmein.org/';
 tkCfg.showDataRelease = true;
 tkCfg.showInfoForPhysicians = false;
-tkCfg.showBlog = true;
+tkCfg.showBlog = false;
 tkCfg.blogUrl = 'http://mbc-project.blogspot.com/';
```

Comment:
I didn't delete blog related entries like url and stuff, just not displaying it. 
May be we want to display it back when the blog is updated !!